src/selectors/balances.js

```diff
@@ -38,28 +38,34 @@ import type {
   ChainTotalBalancesPerAccount,
   CategoryTotalBalancesPerChain,
   TotalBalancesPerChain,
-  AssetsBalances,
+  WalletAssetsBalances,
+  CategoryBalancesPerChain,
+  AssetBalancesPerAccount,
 } from 'models/Balances';
 
 // selectors
 import {
-  balancesSelector,
+  assetsBalancesSelector,
   fiatCurrencySelector,
   ratesSelector,
   activeAccountIdSelector,
   activeAccountSelector,
 } from './selectors';
 
-
-export const accountEthereumWalletBalancesSelector = createSelector(
-  balancesSelector,
+export const accountAssetsBalancesSelector = createSelector(
+  assetsBalancesSelector,
   activeAccountIdSelector,
-  (balances, activeAccountId): AssetsBalances => {
+  (balances: AssetBalancesPerAccount, activeAccountId: ?string): CategoryBalancesPerChain | {} => {
     if (!activeAccountId) return {};
-    return balances?.[activeAccountId]?.[CHAIN.ETHEREUM]?.[ASSET_CATEGORY.WALLET] || {};
+    return balances?.[activeAccountId] ?? {};
   },
 );
 
+export const accountEthereumWalletAssetsBalancesSelector = createSelector(
```

Comment:
Seems like this selector returning WalletAssetBalances is used as a drop in replacement for AssetBalances type. But the new types stores values a BigNumber while previous one used string. 
1. Pls verify that the affects screens work fine
2. Pls add typing for selector, because createSelectors returns any.
3. 
```suggestion
import type { Selector } from 'reduces/rootReducer';
///.... 
export const accountEthereumWalletAssetsBalancesSelector: Selector<WalletAssetsBalances> = createSelector(
```

Reply:
It does, cannot find where it actually returned BigNumber? It used to be `Balances` type which was always string for `balance` 🤔 screens work fine.

Reply:
Ah I see, it expects `BigNumber` on Assets screen for Wallet category.

Reply:
For that `useWalletAssets` wraps it into `BigNumber` before it reaches UI.